java/ql/src/Security/CWE/CWE-489/DebuggableAttributeEnabled.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Android debuggable attribute enabled
+ * @description An enabled debugger can allow for entry points in the application or reveal sensitive information.
+ * @kind problem
+ * @problem.severity warning
+ * @security-severity 7.2
+ * @id java/android/debuggable-attribute-enabled
+ * @tags security
+ *       external/cwe/cwe-489
+ * @precision very-high
+ */
+
+import java
+import semmle.code.xml.AndroidManifest
+
+from AndroidXmlAttribute androidXmlAttr
+where
+  androidXmlAttr.getName() = "debuggable" and
+  androidXmlAttr.getValue() = "true" and
+  not androidXmlAttr.getLocation().getFile().getRelativePath().matches("%build%")
```

Comment:
Maybe this is worth a comment about why we're making this exclusion, optionally linking documentation for more detail: https://developer.android.com/studio/build/manage-manifests

Reply:
Library updated with predicates in commit [171a47e](https://github.com/github/codeql/pull/9939/commits/171a47e816feaeda3533f793510eb12bdc76bf0f).
@atorralba Let me know if I should include a more detailed QLDoc comment on the `isInBuildDirectory()` predicate of the `AndroidManifestXmlFile` class.